CMSgov/design-system
# [WNMGDS-642] Add USWDS spacing design tokens

## Summary
This PR adds USWDS spacing design tokens for margin/padding in the design system. 
Note: USWDS spacing tokens are documented in px, but they output values in rem which may break some sass calculation using mixed units of 'px' and 'rem' as a property value, which will produce an incompatible error. The fix is to change these property value to USWDS spacing token via the `units()` function

### Added
- Add USWDS as dependency
- Add USWDS sass and theme setting

### Changed
- Update the places where padding and margin is declared in the CMS Design System to use USWDS spacing design tokens

## How to test
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-642/use-uswds-spacing-design-tokens/

- Test this branch on both the Mgov and HCgov child design systems (spacing-token branch) by updating the package.json to point to this branch. Ensure Child DS work as expected.
  - [Hcgov DS test branch: WNMGDS-642/use-uswds-spacing-tokens](https://github.com/CMSgov/hcgov-design-system/tree/WNMGDS-642/use-uswds-spacing-tokens)
  - [Mgov DS test branch: WNMGDS-642/use-uswds-spacing-tokens](https://github.com/CMSgov/mgov-design-system/tree/WNMGDS-642/use-uswds-spacing-tokens)

```
"@cmsgov/design-system": "file:../design-system/packages/design-system",
"@cmsgov/design-system-docs": "file:../design-system/packages/design-system-docs",
"@cmsgov/design-system-scripts": "file:../design-system/packages/design-system-scripts",
```
